app/mailers/user_mailer.rb

```diff
@@ -422,6 +400,17 @@ def open_project_headers(hash)
 # Unfortunately, this results in changes on the interceptor classes during development mode
 # not being reflected until a server restart.
 
+class PlainTextInterceptor
+  def self.delivering_email(message)
+    if Setting.plain_text_mail?
+      # HACK: this is a workaround to ensure @html_part ivar is set correctly
+      # on the Mail::Message.
+      message.html_part = message.html_part
+      message.html_part = nil
```

Comment:
@ulferts that would make the most sense to me too. However, the comment on the method that 78e8fc38e3faa94114f83213490f390378126f4d eliminates, seems to indicate that that would be problematic too.